### ISSUE TITLE
training time stats

### ISSUE THREAD

**Issue by shivanimall** [google-deepmind/tapnet#98]
I am using TAPIR as backbone architecture, and starting from the checkpoint version. I have to further train it based on changes. Do you have any suggestions for what training times to expect from the checkpoint version? or have any fine-tuning stats? 

For example, it seems that BootTAP uses pre-trained TAPIR as backbone, I wonder what were the training time stats for it, in case I missed on the paper. 

Thank you!

**Comment by cdoersch** [comment]
Sorry, we haven't been releasing much on the training loss curves because it's difficult to maintain them.  BootsTAPIR took about 2 weeks to train on YouTube data on 256 A100 GPUs, but that's due to the diversity of YouTube.  Further finetuning it on Libero took about 3 days (50K steps) on 128 GPUs, but it only took that long because we were jointly training on youtube and kubric as well.  I suspect that a much shorter training run will produce similar results, but we haven't done rigorous experiments.

Unfortunately we haven't yet released BootsTAP training code, and it's unclear when we'll be able get to it considering how busy the team is.  I'm hoping it'll be before ECCV, but I can't make any guarantees.

**Comment by shivanimall** [comment]
thank you for sharing these! I will leave this issue open, as I may follow back here with further questions, plus share my training time stats results as well, and in case that helps anyone.

**Comment by Kou-99** [comment]
> Sorry, we haven't been releasing much on the training loss curves because it's difficult to maintain them. BootsTAPIR took about 2 weeks to train on YouTube data on 256 A100 GPUs, but that's due to the diversity of YouTube. Further finetuning it on Libero took about 3 days (50K steps) on 128 GPUs, but it only took that long because we were jointly training on youtube and kubric as well. I suspect that a much shorter training run will produce similar results, but we haven't done rigorous experiments.
> 
> Unfortunately we haven't yet released BootsTAP training code, and it's unclear when we'll be able get to it considering how busy the team is. I'm hoping it'll be before ECCV, but I can't make any guarantees.

Hi, is there any update on the release time of the BootsTAP training code?

**Comment by yangyi02** [comment]
Hi @Kou-99 , thanks for your interest! At the moment, we don’t have plans to release the BootsTAP training code since it is tightly integrated with Google’s internal systems.
